m2m-transfers/V1/m2m-transfers-json-contract.md

```diff
@@ -333,35 +333,35 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
+*See Modifier section for additional details
 
-| Error Code | Logging Error Message                                                       | Notes |
-| -----------|------------------------------------------------------------------------------------- |-----------------------------------------------|
-| 500        | Program running in memo mode ||
-| 501        | Config file error: [configuration file name] open error - [system generated letter file read error message] | This error represents issues with the configuration letter file setup |
-|            | Config file error: [configuration file name] read error - [system generated letter file read error message] ||
-|            | Config file error: 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
-|            | Config file error: Invalid Parameter in CFG file ||
-| 502        | Invalid Source Account: Pref Access type 3 not found ||
-|            | Invalid Source Account: Acct Warning 1234 ||
-|            | Invalid Source Account: No eligible transfer from shares/loans ||
-|            | Invalid Source Account: Acct Type 1234 ||
-| 503        | Invalid Recipient Account ||
-| 504        | Invalid Source AccountCannot calc member limits. ||
-| 505        | Invalid Input ||
-| 506        | Error Processing Recipient Record | Error creating/deleting recipient |
-| 507        | Error Processing Transfer Record | Error creating/updating/deleting transfer |
-|            | Error Processing Transfer RecordTarget s/l xfer Loc 1234567 not found ||
-|            | Error Processing Transfer Record: Target ID not found or invalid ||
-| 508        | Undefined Error ||
-| 509        | Member verification failed | Member unverified |
-|            | Member verification failed: Account Closed - 99/99/99 ||
-|            | Member verification failed: Name verification failed ||
-|            | Member verification failed: No valid share or loan found ||
-|            | Member verification failed: Target S/L not found ||
-|            | Member verification failed: S/L closed or charged-off - 99/99/99 ||
-| 510        | Account ID incorrect ||
-| 511        | Request exceeds limits ||
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
+| 501        | Config file error | : [configuration file name] open error - [system generated letter file read error message] | This set of errors represents issues with the configuration letter file setup |
```

Comment:
Isn't the "This set of errors represents issues with the configuration letter file setup" notation in col. 4 already stated by what's in col. 2 - "Config file error"?  the same with 'This set of errors represents issues with the source account' and in col.1  - 'Invalid Source Account', etc.